packages/react-examples/.storybook/preview.js

```diff
@@ -10,24 +10,28 @@ addDecorator(withPerformance);
 addDecorator(withInfo());
 addDecorator(withKeytipLayer);
 if (
-  [
-    'react-button',
-    'react-cards',
-    'react-checkbox',
-    'react-link',
-    'react-slider',
-    'react-tabs',
-    'react-text',
-    'react-toggle',
-  ].includes('PACKAGE_NAME')
+  ['react-cards', 'react-checkbox', 'react-link', 'react-slider', 'react-tabs', 'react-text', 'react-toggle'].includes(
+    'PACKAGE_NAME',
+  )
 ) {
+  initializeIcons();
   addDecorator(withCompatThemeProvider);
   addDecorator(withStrictMode);
 }
 if (
-  ['react-avatar', 'react-badge', 'react-link', 'react-image', 'react-menu', 'react-components'].includes(
-    'PACKAGE_NAME',
-  )
+  [
+    'react-avatar',
+    'react-badge',
+    'react-components',
+    'react-button',
+    'react-link',
+    'react-image',
+    'react-avatar',
+    'react-badge',
+    'react-link',
+    'react-image',
+    'react-menu',
```

Comment:
We should remove duplicates here:
```suggestion
    'react-avatar',
    'react-badge',
    'react-button',
    'react-components',
    'react-image',
    'react-link',
```

Reply:
Bad merge! Thanks for catching :D